### ISSUE TITLE
Word Counts on FireFox

### ISSUE THREAD

**Issue by adreeve** [w8tcha/CKEditor-WordCount-Plugin#58]
The strip function in this plugin seems questionable to me... in particular the following:

```
if (tmp.textContent == '' && typeof tmp.innerText == 'undefined') {
    return '0';
}
return tmp.textContent || tmp.innerText;
```

If the editor's content is empty then the if clause will be true and thus the string '0' will be returned which should have a wordcount of 1 despite the editor being empty and having 0 words in it. It seems to me this whole thing should be replaced by:

```
return tmp.textContent || tmp.innerText || '';
```

so FF returns '' if it's empty as other browsers do?

In a nutshell we're trying to deal with getting the right property, but have issues when '' is retrieved as it's falsy too just like undefined is so we need a 'default' return value of '' for when the first property is what we want, but is also falsy and the second is undefined.

Am I wrong?
